core/git.go

```diff
@@ -0,0 +1,101 @@
+package core
+
+import (
+	"context"
+
+	"github.com/dagger/dagger/core/pipeline"
+	"github.com/dagger/dagger/core/socket"
+	"github.com/dagger/dagger/engine"
+	"github.com/dagger/dagger/engine/buildkit"
+	"github.com/dagger/dagger/engine/sources/gitdns"
+	"github.com/moby/buildkit/client/llb"
+	specs "github.com/opencontainers/image-spec/specs-go/v1"
+	"github.com/pkg/errors"
+)
+
+type GitRef struct {
+	URL string `ssh:"url"`
+	Ref string `ssh:"ref"`
+
+	KeepGitDir bool `ssh:"keepGitDir"`
+
+	SSHKnownHosts string    `json:"sshKnownHosts"`
+	SSHAuthSocket socket.ID `json:"sshAuthSocket"`
+
+	Services ServiceBindings `ssh:"services"`
```

Comment:
is ``` `ssh` ``` intentional in the places it's used here?

Reply:
Uhhh. No. Not quite sure what happened there, I'm gonna blame a typo :laughing: